app/frontend/src/views/form/Manage.vue

```diff
@@ -203,15 +217,17 @@ export default {
       this.formSettingsDisabled = true;
       this.fetchForm(this.f);
     },
+    deleteForm() {
+      this.showDeleteDialog = false;
+      this.deleteCurrentForm();
```

Comment:
Do we need to make this an async/await to make sure it's deleted before navigating?

Reply:
No was going to navigate away regardless and the error will pop if it fails. Not losing any work if they have to click back and try again.

Reply:
For the current case it should be fine as all operations get shunted to vuex anyways (error notification, etc). It's more in the case of if it fails to delete, we don't want them to navigate away, and leave them on the Manage page with the error instead.